src/scss/02_atoms/form/_input.scss

```diff
@@ -0,0 +1,49 @@
+@mixin inputState($state) {
+  background: get($formFieldStateConfig, $state, background);
+  border-color: get($formFieldStateConfig, $state, stateColor);
+  box-shadow: get($formFieldStateConfig, $state, boxShadow);
+  color: get($formFieldStateConfig, $state, color);
+  opacity: get($formFieldStateConfig, $state, opacity);
+  -webkit-text-fill-color: get($formFieldStateConfig, $state, color);
+}
+
+.form-input {
+  border-radius: get($formFieldStyleConfig, borderRadius);
+  border-style: solid;
+  border-width: get($formFieldStyleConfig, borderWidth);
+  font-family: get($formFieldStyleConfig, fontFamily);
+  font-size: get($formFieldStyleConfig, fontSize);
+  height: get($formFieldStyleConfig, height);
+  outline: get($formFieldStyleConfig, outline);
+  padding-bottom: get($formFieldStyleConfig, paddingBottom);
+  padding-left: get($formFieldStyleConfig, paddingLeft);
+  padding-right: get($formFieldStyleConfig, paddingRight);
+  padding-top: get($formFieldStyleConfig, paddingTop);
+  transition: background 0.3s, border 0.3s, color 0.3s;
+
+  @include formFieldSizes;
```

Comment:
Includes in testa e senza spazio di separazione

Reply:
Questo include era stato messo in fondo volontariamente. Ema prima di portarlo su fai un controllo, sono abbastanza certo che sia stato messo in fondo per un motivo